docs/tutorials/installation-success.md

```diff
@@ -0,0 +1,22 @@
+---
+   description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
```

Comment:
This page is no longer a guide to install AES but rather some suggestions on next steps once it is installed. It's also shared with the main install page, so maybe consider moving it under the install folder and having both pages (the getting started overlay and the main install page) reference it?